index.html

```diff
@@ -0,0 +1,169 @@
+            <option value="3">Fulltime Learner</option>
+          </select>
+        </div>
+
+        <div class="form-check">
+          <p>Would you recommend freeCodeCamp to a friend?</p>
+
+          <input
+            class="form-check-input"
+            type="radio"
+            name="flexRadioDefault"
+            id="flexRadioDefault1"
+            value="def"
+          />
+          <label class="form-check-label" for="flexRadioDefault1">
+            Definitely
+          </label>
+        </div>
+        <div class="form-check">
+          <input
+            class="form-check-input"
+            type="radio"
+            name="flexRadioDefault"
+            id="flexRadioDefault2"
+            checked
+            value="may"
+          />
+          <label class="form-check-label" for="flexRadioDefault2">
+            Maybe
+          </label>
+        </div>
+        <div class="form-check">
+          <input
+            class="form-check-input"
+            type="radio"
+            name="flexRadioDefault"
+            id="flexRadioDefault3"
+            value="not"
+          />
+          <label class="form-check-label" for="flexRadioDefault3">
+            Not Sure
+          </label>
+        </div>
+
+        <div class="form-check">
+          <p>What would you like to see improved? (Check all that apply)</p>
+          <input
+            class="form-check-input"
+            type="checkbox"
+            value="fep"
+            id="flexCheckDefault"
+          />
+          <label class="form-check-label" for="flexCheckDefault">
+            Front-end projects
+          </label>
+        </div>
+        <div class="form-check">
+          <input
+            class="form-check-input"
+            type="checkbox"
+            value="bep"
+            id="flexCheckChecked"
+            checked
+          />
+          <label class="form-check-label" for="flexCheckChecked">
+            Back-end projects
+          </label>
+        </div>
+        <div class="input-group">
+          <span class="input-group-text">With textarea</span>
+          <textarea class="form-control" aria-label="With textarea"></textarea>
+        </div>
+
+        <button type="submit" class="btn btn-primary" id="submit">
+          Submit
+        </button>
+      </form>
+    </section>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa"
+      crossorigin="anonymous"
+    ></script>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Do not leave comments that are not directly relevant to the code maintenance or further action in your code base, this is not a professional best practice

Reply:
- Great Job using Inline comments ✔️ 

[OPTIONAL] 👍🏽 
Kindly remember to use emojis in the inline comments just as @nicupop729 advised 🍷